src/contract_client/sent_transaction.ts

```diff
@@ -118,7 +133,7 @@ export class SentTransaction<T> {
       this.getTransactionResponse.status ===
       SorobanRpc.Api.GetTransactionStatus.NOT_FOUND
     ) {
-      console.error(
+      throw new SentTransaction.Errors.TransactionStillPending(
```

Comment:
Does this make the change a breaking change? If so we might need to wait for the next release?

Reply:
Or not actually because the release is an rc.

Reply:
yeah an rc makes it kosher but also `ContractClient` isn't published by the SDK so it's also kosher there
it'd be a breaking change for the cli/ts bindings but that is still safe because of the rc

Reply:
Yeah, the linter didn't like the `console` statement, which made me stop and think—do we actually want it to just return, with just a `console.error` statement, if things are not actually in an expected state? If we're in an unexpected state, then we probably want to throw an error.

Reply:
Should this be noted in the changelog too?

Reply:
It has now been noted in the changelog